README.md

```diff
@@ -7,8 +7,16 @@ variables, functions and methods. Within each of these categories, exported decl
 
 ## Installation
 
+### go 1.16+
+
+```sh
+go install github.com/jdeflander/goarrange@v1.0.0
+```
+
+### pre go 1.16
+
 ```sh
-$ go get github.com/jdeflander/goarrange
+go get github.com/jdeflander/goarrange
```

Comment:
I removed it here to make copy-pasting easier (yeah i'm that lazy), but for consistency it's probably better to readd the `$` everywhere then because otherwise you won't be able to make that distinction in the usage section.